app/service/auth_svc.py

```diff
@@ -118,13 +145,19 @@ def request_has_valid_api_key(self, request):
     async def request_has_valid_user_session(self, request):
         return await aiohttp_security_api.authorized_userid(request) is not None
 
+    async def provide_verified_login_response(self, request, username):
+        self.log.debug('%s logging in:', username)
```

Comment:
Trailing colon looks maybe out of place?

Reply:
Hmm I think that was always there - I'll remove it, though